save-frontend/src/main/resources/highlight-css/intellij-light.css

```diff
@@ -0,0 +1,117 @@
+/*
+Intellij-light style (c) Pegasis <me@pegasis.site>
```

Comment:
Why do we need code, when it's loaded as `<link>`?

Reply:
Oh, will replace local css with css from cloudflare